### PR TITLE
Adding gamerule.txt configuration file handling (optional)

### PR DESCRIPTION
**Pull request by nlloyd**
I wanted to ensure that certain "/gamerule" settings were established on server startup. I made a simple change to feed in a gamerules.txt file in start-minecraft.sh, if the file is found in the /data directory.

### CODE REVIEW

File: minecraft-server/start-minecraft.sh [hunk 1]
```diff
@@ -253,5 +253,10 @@ do
   fi
 done
 
-
-exec java $JVM_OPTS -jar $SERVER
+# If we have a gamerules.txt file... feed that in to the server stdin
+if [ -f /data/gamerules.txt ];
```
Comment (itzg) [inline minecraft-server/start-minecraft.sh]: Thanks for contribution.

How about you have it look for something more generic, like `/data/bootstrap.txt`, to expand the semantics of how this be used.

**Comment by itzg** [comment]
@nlloyd , I'd like to get this in. Can you resolve the merge conflict? But let me know if you have any questions/concerns -- from the web interface it's not showing me the actual conflict.

**Comment by nlloyd** [comment]
Done! Excellent idea on bootstrap.txt.

**Comment by itzg** [comment]
Your addition has been pushed to Hub as image ID 12e93f1cd21c. Thanks again.

**Pull request merged**